pandas/_libs/tslibs/strptime.pyx

```diff
@@ -597,7 +636,23 @@ cdef _calc_julian_from_U_or_W(int year, int week_of_year,
                               int day_of_week, int week_starts_Mon):
     """Calculate the Julian day based on the year, week of the year, and day of
     the week, with week_start_day representing whether the week of the year
-    assumes the week starts on Sunday or Monday (6 or 0)."""
+    assumes the week starts on Sunday or Monday (6 or 0).
```

Comment:
can you type the return here (cdef int)

Reply:
Sorry I don't understand what you mean. In the description of the summary or something else?

Reply:
no the return type of the  function (guess it wasn't typed before)

```
cdef int _calc_julian_from_U_or_W(......)
```

also you can type the one right below as ``cdef object .....``